src/Microsoft.ApplicationInsights/Managed/Shared/Extensibility/Implementation/Tracing/BaseHeartbeatProperties.cs

```diff
@@ -0,0 +1,181 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Tracing
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Reflection;
+    using System.Text;
+    using System.Threading.Tasks;
+#if NETSTANDARD1_3
+    using System.Runtime.InteropServices;
+#endif
+
+    internal class BaseHeartbeatProperties : IHeartbeatDefaultPayloadProvider
```

Comment:
name of class not really indicative of the interface it implements.

Reply:
Changed!